scope-on-docker-for-mac/README.md

```diff
@@ -0,0 +1,43 @@
+### Install Docker for Mac
+
+If you haven't installed Docker for Mac, please follow the installation instructions on [Docker website][install-d4m].
+
+Once it's runing you should see <img alt="Docker Icon in the Mac OS menu bar" src="docker-for-mac-menu-bar-icon.png"
+style="height: 1em;" /> in your menu bar.
+
+[install-d4m]: https://docs.docker.com/docker-for-mac/
+
+### Install the Application
+
+Get the code:
+```
+git clone https://github.com/weaveworks/guides
+git checkout weave-demo-app
+cd scope-on-docker-for-mac
+```
+Deploy the shop app:
+```
+docker-compose pull
+docker-compose -p shop up -d && open http://localhost:80
+```
+
+### Signup to Weave Cloud
+
+1. Go to ***[cloud.weave.works](https://cloud.weave.works)***
+2. Obtain service token as shown below
+
+![Obtain service token for Weave Cloud](weave-cloud-token-screenshot.png)
+
+### Use Weave Cloud
```

Comment:
This heading should be "Connect Scope Probe to Weave Cloud" and have some explanatory text explaining what that means and why the user would want to do it.